test/fn/upload-custom-translations.spec.js

```diff
@@ -39,6 +39,7 @@ describe('upload-custom-translations', () => {
         .then(() => expectTranslationDocs('en'))
         .then(() => getTranslationDoc('en'))
         .then(messagesEn => {
+          assert(messagesEn.name === 'English');
```

Comment:
Can I request a unit test which triggers:

1. The case where there is `native (english)` and validate the expected name.
1. An invalid language code and validate the default is set.